src/screens/account/account.tsx

```diff
@@ -66,8 +78,18 @@ export class AccountScreenComponent extends React.Component<
         this.setState({ settingsVisible: !this.state.settingsVisible });
     };
 
+    public firstLine = (tx: ITransactionState, account: IAccountState) => {
```

Comment:
please don't use arrow function in classes

Reply:
also try to use more suggestive names, and if method returns something maybe use a `get` prefix.  Like `getTransactionPrimaryText()`